.travis.yml

```diff
@@ -9,8 +9,6 @@ notifications:
 node_js:
   - '6'
   - '4'
-  - '0.12'
```

Comment:
By dropping `Node<4` we will of course be forced to drop support in the CLI when we update to the latest version of `resolve-deps`.

I assume that this is intentional (dropping support in the CLI now).

Reply:
We officially stopped supporting `Node<4` on March 5th 2018

Reply:
Where can I go to see that? I checked the [package.json](https://github.com/snyk/snyk-internal/blob/develop/package.json) but didn't see any `engines` mentioned. I believe we need to release `cli` with an `engines` set to `0.12` first so that users of `Node@0.12` stay on that version.

Reply:
Good point!